src/es/elastic.ts

```diff
@@ -52,35 +50,44 @@ export const getDocumentsFromES = async (
   const docs: any[] = [];
 
   if (withDocs) {
-    const treatResponse = ({ body }: { body: any }) => {
-      // get docs from response
-      body.hits.hits.forEach((hit: any) => {
+    const treatResponse = ({ hits }: { hits: any }) => {
+      hits.forEach((hit: any) => {
         docs.push(hit._source);
       });
+      if (docs.length % 50000 == 0) {
+        logger.debug(docs.length);
+      }
 
-      // return next scroll id
-      return body._scroll_id;
+      return hits[hits.length - 1].sort[0];
```

Comment:
il doit y avoir un rare cas où `hits.length===0`

Reply:
@carolineBda je suis pas sûr qu'on a maj ça. Pour moi il faudrait un truc comme ça avant le return :

```suggestion
return hits.length > 0 ? hits[hits.length - 1].sort[0] : null
```

Reply:
je fais le check juste avant d'appeler la method : https://github.com/SocialGouv/cdtn-monolog/pull/155/files#diff-353024b0cf20bf5435109930f54d7e45ffb602bd3aa53710d86a86002ab6f4e2R87

Reply:
Mais si c'est un tableau vide, ça passe quand même, non ?